src/com/owncloud/android/ui/activity/FileDisplayActivity.java

```diff
@@ -321,6 +322,8 @@ private void initFragmentsWithFile() {
                     startTextPreview(file);
             }
 
+            switchLayout(getFile());
```

Comment:
As it is only a quick change, maybe you can do it, if it otherwise will prevent this PR from QA?
(I can do it on sunday or monday afternoon)